frontend/src/main/components/Chat/ChatMessageCreate.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Button, Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+import { useBackendMutation } from "main/utils/useBackend";
+
+const ChatMessageCreate = ({ commonsId, submitAction: submitProp }) => {
```

Comment:
`onSubmit` is a more idiomatic name for these callbacks.

Reply:
same as above

Reply:
I should be more clear, the submitAction is separate from the onSubmit, as the onSubmit callback for the button is handleSubmit from react-bootstrap Form. submitAction is what we pass into handleSubmit